src/content/docs/en/pages/devtools/cli/azion-cli/config/azion-config-js.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: azion.config.js file
+description: Learn how to customize your application build process using the azion.config.js file. This documentation provides detailed explanations of the configuration properties.
+meta_tags: azion, configuration, javascript, build process
+namespace: documentation_devtools_configs_azion_config_js
+permalink: /documentation/devtools/cli/configs/azion-config-js/
+menu_namespace: cliMenuAlpha
+---
+
+The `azion.config.js` file is a configuration file created during the build of the application, based on the chosen preset. Each preset presents a set of default settings. These settings can be replaced by the user. This file is the source of truth for the configuration. In case it's deleted, when the build process is triggered again, the default configuration will be recreated.
+
+The following tables explain the properties of this configuration file.
+
+---
+
+## origin
+
+| Property | Description | Values | Required |
+| --- | --- | - | - |
+| name | The name of the origin. |  | Yes |
+| type | The type of the origin. In this case, it's an object storage.  |  Possible values: `single_origin` (default)` load_balancer`,`live_ingest` or `object_storage` | Required |
+| bucket | The name of the bucket in the object storage. |  | Yes for object_storage type origins. |
+| prefix | The prefix for the objects in the bucket. |   | No |
+
+[For further information, access Azion API specification](https://api.azion.com/#adbc0ab3-8f27-47e2-a8fe-428c7fa93250)
+
+---
+
+## cache
+
+| Property | Description |  Required |
+| --- | --- | --- |
+| name | Name of your cache setting. | Yes|
+| stale | A boolean indicating whether stale content can be served. | |
+| queryStringSort | Considers objects with the same query strings as the same cached file regardless of the order of the fields. | |
+| methods | An object specifying which HTTP methods should be cached. | |
+| browser | An object specifying the maximum age of the cache in the browser. | |
+| edge | An object specifying the maximum age of the cache at the edge. | |
```

Comment:
```suggestion
| name | Name of your cache setting | Yes |
| stale | A boolean indicating whether stale content can be served | |
| queryStringSort | Considers objects with the same query strings as the same cached file regardless of the order of the fields | |
| methods | An object specifying which HTTP methods should be cached | |
| browser | An object specifying the maximum age of the cache in the browser | |
| edge | An object specifying the maximum age of the cache at the edge | |
```

Reply:
You can add "No" in the Required column for non-required items. To align with the first table.